bitcoin-extensions/bitcoin.html

```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
+		<title>Bitcoin Price Index Watcher</title>
+		<link href="styles.css" rel="stylesheet" type="text/css">
+	</head>
+	<body>
+		<img id="logo" src="bitcoin.png" />
+		<div id="data"></div>
+		<script type="text/javascript">
+			var xmlhttp = new XMLHttpRequest();
+			var url = "https://api.coindesk.com/v1/bpi/currentprice.json";
+
+			xmlhttp.onreadystatechange = function() {
+				if (this.readyState == 4 && this.status == 200) {
+					var json = JSON.parse(this.responseText);
+					parseJson(json);
+				}
+			};
+
+			xmlhttp.open("GET", url, true);
+			xmlhttp.send();
+
+			function parseJson(json) {
+				var time = "<b>Last Updated: " + json["time"]["updated"] + "</b>"
+				var usdValue = "1 BTC equals to $" + json["bpi"]["USD"]["rate"];
+				var gbpValue = "1 BTC equals to &pound;" + json["bpi"]["GBP"]["rate"];
+				var euroValue = "1 BTC equals to &euro;" + json["bpi"]["EUR"]["rate"];
+
+				document.getElementById("data").innerHTML = time + "<br /><br />" + usdValue + "<br />" + gbpValue + "<br />" + euroValue;
```

Comment:
...you could attach the different values and times to each div rather than all on the one. Nice API usage, awesome idea